### PR TITLE
feat(k8s launch):Support forwarding unrecognized --args to ENTRY_POINT 

### PR DESCRIPTION
**Pull request by Xiaoming-AMD**
### Summary

This PR updates `examples/run_k8s_pretrain.sh` to support forwarding unrecognized `--key value` arguments directly to the container's `ENTRY_POINT` (e.g., `run_pretrain.sh`), **without injecting them into the ENV JSON**. This enables more flexible control over backend-specific training flags from the CLI.

---

### What’s Changed

- Introduced `EXTRA_ARGS=()` to collect unrecognized `--xxx` arguments.
- Updated argument parsing logic:
  - Known options (like `--cpu`, `--gpu`, `--backend`, etc.) are handled as before and added to `ENV_JSON`.
  - Unknown options are now appended to `EXTRA_ARGS` and **not added to ENV_JSON**.
- Updated `ENTRY_POINT` to include `${EXTRA_ARGS[*]}` so they are passed to `run_pretrain.sh`.
- Added debug print for `EXTRA_ARGS`.
- Updated usage instructions.

---

### Usage Example

```bash
./run_k8s_pretrain.sh --url http://api.example.com create \
    --replica 2 --cpu 96 --gpu 4 \
    --exp examples/megatron/configs/llama2_7B.yaml \
    --data_path /mnt/data/train \
    --workspace team-dev \
    --batch-size 64 --lr 1e-4 --seed 42
```
 - --batch-size, --lr, --seed are passed through to the container’s training script.

### CODE REVIEW

File: examples/megatron/configs/deepseekv2_lite_pretrain.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      wandb_project: "Primus_DeepSeek_Pretrain"
+      # disable_wandb: false
+      # disable_tensorboard: false
+      stderr_sink_level: DEBUG
+
+      # debug
+      # num_layers: 5
+      # optimizer: adam
+      moe_router_force_load_balancing: true
+      moe_router_dtype: fp32
```
Comment (wenxie-amd) [inline examples/megatron/configs/deepseekv2_lite_pretrain.yaml]: 这个地方去掉吧，其他的也去掉，默认bf16。要不会有fp32 reducescatter开销比较大
